include/sensors/core/tasks/pressure_driver.hpp

```diff
@@ -27,10 +28,12 @@ template <class I2CQueueWriter, class I2CQueuePoller,
 class MMR92C04 {
   public:
     MMR92C04(I2CQueueWriter &writer, I2CQueuePoller &poller,
-             CanClient &can_client, OwnQueue &own_queue)
+             CanClient &can_client, hardware::SensorHardwareBase &hardware,
+             OwnQueue &own_queue)
         : writer(writer),
           poller(poller),
           can_client(can_client),
+          hardware(hardware),
```

Comment:
Are you planning on using the data ready pin in a follow-up PR? Otherwise, right now it's not being used.